src/main/java/graphql/schema/GraphqlDirectivesContainerTypeBuilder.java

```diff
@@ -22,6 +21,33 @@ public B replaceAppliedDirectives(List<GraphQLAppliedDirective> directives) {
         return (B) this;
     }
 
+    public B addAppliedDirectives(GraphQLAppliedDirective... directives) {
+        assertNotNull(directives, () -> "directives can't be null");
+        for (GraphQLAppliedDirective directive : directives) {
+            addAppliedDirective(directive);
+        }
+        return (B) this;
+    }
+
+    public B addAppliedDirective(GraphQLAppliedDirective directive) {
+        assertNotNull(directive, () -> "directive can't be null");
+        this.appliedDirectives.add(directive);
+        return (B) this;
+    }
+
+    public B addAppliedDirective(GraphQLAppliedDirective.Builder builder) {
+        return addAppliedDirective(builder.build());
+    }
```

Comment:
Because it makes more sense for all the additive builders to start with "add", I have also created the "add" version for the singular method, but I think the prior implementation was correct